benchmark/Gopkg.toml

```diff
@@ -0,0 +1,11 @@
+[[constraint]]
+  name = "github.com/pkg/errors"
+  version = "0.8.0"
+
+[[constraint]]
+  name = "k8s.io/client-go"
```

Comment:
seems like you cannot resolve this version with Prometheus. Can you ensure you can run `make deps` first?